storybook/src/docs/form/components/NumberField.stories.mdx

```diff
@@ -0,0 +1,27 @@
+import { Meta, Canvas, Story, Source } from "@storybook/addon-docs";
+
+<Meta title="Docs/Form/Components/NumberField" />
+
+# Number Field
+
+The NumberField can be used for number inputs. It automatically formats the numbers with thousand-separators and decimal symbols in the current locale. You can also use it for currency inputs by passing the currency symbol as `startAdornment` or `endAdornment`. The value is stored as type `number` in the form values.
+
+## Basic usage
+
+<Canvas>
+    <Story id="stories-form-components--numberfield" />
+</Canvas>
+
+## Usage with decimals
+
+The decimals prop is used to define, how many decimal digits the value should have.
+
+<Canvas>
+    <Story id="stories-form-components--withdecimals" />
+</Canvas>
+
+## Usage as Currency Field
+
+<Canvas>
+    <Story id="stories-form-components--currency" />
+</Canvas>
```

Comment:
The stories are not loading anymore, because the id changed. 

```suggestion
<Canvas>
    <Story id="stories-form-components-numberfield--numberfield" />
</Canvas>

## Usage with decimals

The decimals prop is used to define, how many decimal digits the value should have.

<Canvas>
    <Story id="stories-form-components-numberfield--withdecimals" />
</Canvas>

## Usage as Currency Field

<Canvas>
    <Story id="stories-form-components-numberfield--currency" />
</Canvas>
```